### DISCUSSION TITLE
Impossible d'installer eleventy

### DISCUSSION THREAD

**Discussion by SoiFranc** [11ty/eleventy#2265] [Q&A]
Bonjour j'ai un problème, je ne peut installer Eleventy avec mon npm.
Je cherche depuis 2jours mais je n'y arrive pas.
Même en regardant : https://github.com/visionmedia/debug/issues/797, 

Avez-vous une solution simple afin de pouvoir installer eleventy, **merci de votre aide**.

Quelques Informations :

npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environm
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environm
ent. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environm
ent. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)

added 348 packages, and audited 349 packages in 2m

25 packages are looking for funding

4 high severity vulnerabilities

# npm audit report

engine.io  <4.0.0
Severity: high
Resource exhaustion in engine.io  - https://github.com/advisories/GHSA-j4f2-536g-r55m
fix available via `npm audit fix --force`
node_modules/browser-sync/node_modules/engine.io
  socket.io  1.0.0-pre - 2.4.1
  Depends on vulnerable versions of engine.io
  node_modules/browser-sync/node_modules/socket.io
    browser-sync  >=1.0.0
    Depends on vulnerable versions of socket.io
    node_modules/browser-sync
      @11ty/eleventy  >=0.3.4
      Depends on vulnerable versions of browser-sync
      node_modules/@11ty/eleventy

4 high severity vulnerabilities

**Comment by pdehaan** [comment]
> Hello I have a problem, I can't install Eleventy with my npm.
> I've been looking for 2 days but I can't find it.
> Even looking at: debug-js/debug#797,
>
> Do you have a simple solution in order to be able to install eleventy, thank you for your help.
>
> Some information :
> &hellip;

What OS and version of Node.js are you using?
What version of @11ty/eleventy are you trying to install? Latest/1.0.0?
Are you trying to install it locally or globally?
I see "added 348 packages, and audited 349 packages in 2m" in the output above. Are you sure Eleventy isn't installing versus just showing some potentially vulnerable dependencies? What happens when you try and build an Eleventy site?

**Comment by SoiFranc** [comment]
Hello,
thanks for your answer
Node.js : v16.14.0
@11ty/eleventy : v1.0.0
"Are you sure Eleventy isn't installing versus just showing some potentially vulnerable dependencies?" Yes, i think so.
"What happens when you try and build an Eleventy site?" Now, it's OK. I've delete node and the install of @11ty/eleventy and i've install new time the both.
i've again the warning but this is ok.
Thank you for your help.
